### DISCUSSION TITLE
How refresh activity in Elsa 3 design

### DISCUSSION THREAD

**Discussion by leostone68** [elsa-workflows/elsa-core#5668] [Q&A]
It's possible refreshing the view of activity and your input field in designer?

**Comment by glime-ai** [comment]
Yes, it is possible to refresh the view of an activity and its input fields in the Elsa Workflow designer. However, as of the information available up to early 2023, there is a known issue where the designer does not automatically refresh certain UI elements, such as dropdown lists using Options providers. This is something that has been recognized by the community and there are discussions on implementing a refresh mechanism, potentially including a refresh button to update the options manually.

For more dynamic updates, there are proposals to use SignalR to push updates from the server to the client, which would allow for real-time updates of the workflow instances and activities within the designer.

If you are experiencing issues with refreshing the view or input fields, you may need to manually trigger a refresh or look into the current state of the Elsa Workflow GitHub issues and discussions to see if there have been any updates or workarounds provided by the community.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
